lsp-mode.el

```diff
@@ -5056,6 +5056,15 @@ When language is nil render as markup if `markdown-mode' is loaded."
       (t (error "Failed to handle %s" content)))
      "")))
 
+(defun lsp--create-unique-sting-fn ()
+  (let (elements)
+    (lambda (element)
+      (let ((count (cl-count element elements :test #'string=)))
+        (prog1 (if (zerop count)
+                   element
+                 (format "%s(%s)" element count))
```

Comment:
nit: since code action title is a normal sentence with spaces, would it make sense to add a space before `(`?
```suggestion
                 (format "%s (%s)" element count))
```